Message/AndroidMessage.php

```diff
@@ -63,7 +63,7 @@ class AndroidMessage implements MessageInterface
      *
      * @param $message
      */
-    public function setMessage($message)
+    public function setMessage($title = null, $message)
```

Comment:
This is a BC break, as it means that all users will have to supply a title. In addition, it is not possible to have an optional parameter in a method signature before any non-optional parameters. This is also on the `AndroidMessage` class, rather than on the iOS one (which is the only place it is used)